pymagicc/io.py

```diff
@@ -415,8 +472,54 @@ def _read_magicc6_style_header(self, stream, metadata):
 
         return column_headers, metadata
 
+    def _get_column_headers_update_metadata(self, stream, metadata):
```

Comment:
What does this do? Get headers or update metadata?

Reply:
both haha